samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.h

```diff
@@ -10,11 +10,12 @@
 
 #define CID_ZEPHYR 0x0002
 
-enum lightness { ONOFF = 0x01, LEVEL, ACTUAL, LINEAR, CTL, IGNORE = 0xFF };
-enum temperature { ONOFF_t = 0x01, LEVEL_t, CTL_t, IGNORE_t = 0xFF };
+enum lightness { ONPOWERUP = 0x01, ONOFF, LEVEL, ACTUAL, LINEAR, CTL, IGNORE};
+enum temperature { ONOFF_t = 0x01, LEVEL_t, CTL_t, IGNORE_t};
```

Comment:
Unrelated to this patch since it already slipped through in the original code, however could you rename the _t suffix to something else? That's normally used for typedefs in Zephyr which makes it quite confusing. Also, keep the names all capital letters. Perhaps `LEVEL_TEMP` or something like that?

Reply:
I will change name to ONOFF_TEMP. And it is related to state binding. 

I will add details in descriptions.

Reply:
Now changed 
enum temperature { ONOFF_t = 0x01, LEVEL_t, CTL_t, IGNORE_t}; to
enum temperature { ONOFF_TEMP = 0x01, LEVEL_TEMP, CTL_TEMP, IGNORE_TEMP};